gdk-pixbuf/src/auto/pixbuf.rs

```diff
@@ -72,6 +72,68 @@ impl Pixbuf {
     //    unsafe { TODO: call ffi:gdk_pixbuf_new_from_data() }
     //}
 
+    #[doc(alias = "gdk_pixbuf_new_from_file")]
+    #[doc(alias = "new_from_file")]
+    pub fn from_file(filename: impl AsRef<std::path::Path>) -> Result<Option<Pixbuf>, glib::Error> {
```

Comment:
```suggestion
    pub fn from_file(filename: impl AsRef<std::path::Path>) -> Result<Pixbuf, glib::Error> {
```

Etc. So these need to stay around in `Gir.toml` and have their return value marked as `nullable = false`, and also have this fixed in gdk-pixbuf to remove the `nullable` annotation on the return value.